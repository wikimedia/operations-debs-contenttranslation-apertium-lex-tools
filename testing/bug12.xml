<rules>
  <rule>
    <match lemma="son" tags="pron.*"><select lemma="dÃ¯hte" tags="pron.*"/></match>
  </rule>
</rules>
