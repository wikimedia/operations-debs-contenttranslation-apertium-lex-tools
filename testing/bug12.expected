^Son<pron><pers><du3><nom>/Dïhte<pron><pers><du3><nom>$ ^ii<v><iv><neg><ind><du3>/ij<v><neg><ind><du3>$ ^sáhttit<v><iv><der_nomag><n><sg><acc><pxsg1>/buektiehtidh<v><tv><der_nomag><n><sg><acc><pxsg1>$ ^stoahka<n><sg><nom><pxsg2>/stååkedimmie<n><sg><nom><pxsg2>$ ^dalle<adv>/joem<adv>$ ^olgun<adv>/ålkene<adv>$^..<clb>/..<clb>$
